ci/build.yml

```diff
@@ -183,12 +183,12 @@ jobs:
           //ledger/... \
           //ledger-service/... \
           //triggers/... \
+          //daml-script/... \
```

Comment:
Was it deliberate to add `//daml-script/` twice?

Reply:
The upper one is the `bazel build` step, the lower one is the `bazel test` step. Arguably those should be factored out but given that I’d like to get this to just `//...`, I couldn’t be bothered to do that so far.

Reply:
Oh right, missed that as the change in command was in the collapsed lines.